crates/bevy_core_pipeline/src/prepass/mod.rs

```diff
@@ -270,3 +292,35 @@ impl CachedRenderPipelinePhaseItem for AlphaMask3dPrepass {
         self.key.pipeline
     }
 }
+
+pub fn prepass_target_descriptors(
+    normal_prepass: bool,
+    motion_vector_prepass: bool,
+    deferred_prepass: bool,
+) -> Vec<Option<ColorTargetState>> {
+    vec![
+        normal_prepass.then_some(ColorTargetState {
+            format: NORMAL_PREPASS_FORMAT,
+            // BlendState::REPLACE is not needed here, and None will be potentially much faster in some cases.
+            blend: None,
+            write_mask: ColorWrites::ALL,
+        }),
+        motion_vector_prepass.then_some(ColorTargetState {
+            format: MOTION_VECTOR_PREPASS_FORMAT,
+            // BlendState::REPLACE is not needed here, and None will be potentially much faster in some cases.
+            blend: None,
+            write_mask: ColorWrites::ALL,
+        }),
+        deferred_prepass.then_some(ColorTargetState {
+            format: DEFERRED_PREPASS_FORMAT,
+            // BlendState::REPLACE is not needed here, and None will be potentially much faster in some cases.
+            blend: None,
+            write_mask: ColorWrites::ALL,
+        }),
+        deferred_prepass.then_some(ColorTargetState {
+            format: DEFERRED_LIGHTING_PASS_ID_FORMAT,
+            blend: None,
```

Comment:
The comment

```rust
// BlendState::REPLACE is not needed here, and None will be potentially much faster in some cases.
```

is missing on this one, intentional?

Reply:
Unsure. @pcwalton?

Reply:
I just copy and pasted that code from elsewhere. Honestly I'm in favor of just removing all those "`BlendState::REPLACE` is not needed..." comments. Blending isn't enabled because there's no need to blend; it's kind of obvious.